col/src/main/java/vct/col/ast/print/PVLPrinter.java

```diff
@@ -598,23 +597,32 @@ public void visit(Contract contract) {
                 e.accept(this);
                 out.lnprintf(";");
             }
-            for(ASTNode e:ASTUtils.conjuncts(contract.pre_condition,StandardOperator.Star)){
-                out.printf("requires ");
-                nextExpr();
-                e.accept(this);
-                out.lnprintf(";");
+            List<ASTNode> contextElems = new ArrayList<>();
```

Comment:
Since this implementation is the same as in JavaPrinter, maybe put this in abstractprinter, the superclass? Then the JavaPrinter only has to put the /*@ @*/ around I think.

Reply:
visit(Contract contract) is now a method in AbstractPrinter, and JavaPrinter adds the @'s and indents to it.